readalongs/cli.py

```diff
@@ -66,116 +73,132 @@ def cli():
-@click.argument('inputfile', type=click.Path(exists=True, readable=True))
-@click.argument('wavfile', type=click.Path(exists=True, readable=True))
-@click.argument('output-base', type=click.Path())
-@click.option('-b', '--bare', is_flag=True, help='Bare alignments do not split silences between words')
-@click.option('-c', '--closed-captioning', is_flag=True, help='Export sentences to WebVTT and SRT files')
-@click.option('-d', '--debug', is_flag=True, help='Add debugging messages to logger')
-@click.option('-f', '--force-overwrite', is_flag=True, help='Force overwrite output files')
-@click.option('-i', '--text-input', is_flag=True, help='Input is plain text (assume paragraphs separated by blank lines, 1 paragraph per page)')
-@click.option('-l', '--language', type=click.Choice(LANGS, case_sensitive=False), help='Set language for plain text input')
-@click.option('-u', '--unit', type=click.Choice(["w", "m"], case_sensitive=False), help='Unit (w = word, m = morpheme) to align to')
-@click.option('-s', '--save-temps', is_flag=True,
-              help='Save intermediate stages of processing and temporary files (dictionary, FSG, tokenization etc)')
-@click.option('-t', '--text-grid', is_flag=True, help='Export to Praat TextGrid & ELAN eaf file')
-@click.option('-x', '--output-xhtml', is_flag=True, help='Output simple XHTML instead of XML')
+@app.cli.command(context_settings=CONTEXT_SETTINGS, short_help="Force align a text and a sound file.")
+@click.argument("inputfile", type=click.Path(exists=True, readable=True))
+@click.argument("audiofile", type=click.Path(exists=True, readable=True))
+@click.argument("output-base", type=click.Path())
+@click.option("-b", "--bare", is_flag=True, help="Bare alignments do not split silences between words")
+@click.option("-c", "--config", type=click.Path(exists=True), help="Use ReadAlong-Studio configuration file")
+@click.option("-C", "--closed-captioning", is_flag=True, help="Export sentences to WebVTT and SRT files")
+@click.option("-d", "--debug", is_flag=True, help="Add debugging messages to logger")
+@click.option("-f", "--force-overwrite", is_flag=True, help="Force overwrite output files")
+@click.option("-i", "--text-input", is_flag=True, help="Input is plain text (assume paragraphs separated by blank lines, 1 paragraph per page)")
+@click.option("-l", "--language", type=click.Choice(LANGS, case_sensitive=False), help="Set language for plain text input")
+@click.option("-u", "--unit", type=click.Choice(["w", "m"], case_sensitive=False), help="Unit (w = word, m = morpheme) to align to")
+@click.option("-s", "--save-temps", is_flag=True, help="Save intermediate stages of processing and temporary files (dictionary, FSG, tokenization etc)")
+@click.option("-t", "--text-grid", is_flag=True, help="Export to Praat TextGrid & ELAN eaf file")
+@click.option("-x", "--output-xhtml", is_flag=True, help="Output simple XHTML instead of XML")
```

Comment:
I see, here and elsewhere, that you standardized strings to double quotes. I searched and found online some advice saying this is fairly common convention, but do we have a style guide we're trying to follow?
Since Python doesn't care about single or double quotes, I have not been consistent about them myself, but I would be happy to have a follow a style guide on this.

Reply:
It doesn't matter, and it's not enforced by pep8. I use double quotes just to be consistent, but I think ideally what we should do is commit to an auto-formatting tool like (black)[https://black.readthedocs.io/en/stable/the_black_code_style.html] and all agree on using that. Or adding a commit hook to run the formatting on before pushing or something.